Neos.ContentRepository.Core/Classes/ContentRepository.php

```diff
@@ -94,6 +94,7 @@ public function __construct(
      */
     public function handle(CommandInterface $command): CommandResult
     {
+        $this->getWorkspaceFinder()->disableRuntimeCache();
```

Comment:
todo might be obsolete due to markStale interface and auto invalidation

Reply:
obsolete and removed